components/constellation/constellation.rs

```diff
@@ -2733,6 +2740,9 @@ where
         if let Err(e) = storage_ipc_receiver.recv() {
             warn!("Exit storage thread failed ({:?})", e);
         }
+        if let Err(e) = webgl_threads_receiver.recv() {
```

Comment:
@gterzian If I understand this correctly, this will be blocking until it receives a value from `webgl_threads_sender`, which happens after `webgl_threads.exit()` has succeeded, right?

Reply:
Yes but see below.